doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
 DOOMGRADER_ROOT=~/doomgrader
 STEAM_PATH=~/.steam/steam/steamapps/common
 
 # internal paths
 DOWNLOAD_PATH=$DOOMGRADER_ROOT/files
 DEPOTDOWNLOADER_PATH=$DOOMGRADER_ROOT/depotdownloader
 
-# prompt for steam credentials
+# function for copying files with prompt
+DOOMGRADER_COPY()
+{
+# copy game files to steam dir
+echo "Copying the game from $DOWNLOAD_PATH to $STEAM_PATH/DOOMEternal/"
+echo "THIS WILL OVERWRITE YOUR EXISTING DOOM ETERNAL GAME FILES!"
+read -p "Is this correct (y/n)?: " DOOMGRADER_ANSWER
+
+if [[ $DOOMGRADER_ANSWER == 'Y' || $DOOMGRADER_ANSWER == "yes" || $DOOMGRADER_ANSWER == "YES" || $DOOMGRADER_ANSWER == "y" || $DOOMGRADER_ANSWER == "Yes" ]]
+then
+    cp $DOWNLOAD_PATH/* $STEAM_PATH/DOOMEternal/ -rfv
+else
+    echo "Copying of files stopped!"
```

Comment:
Please remove the exclamation marks here. The wording is good!

Reply:
Sure thing!